base_login/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+# Create your models here.
+
+class ActivityTypes(models.Model):
+    name = models.CharField(max_length=255)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return f'{self.name}'
+
+class Activities(models.Model):
+    name = models.CharField(max_length=255)
+    type = models.ForeignKey('ActivityTypes', on_delete=models.CASCADE)
```

Comment:
I'm not sure we'd want an `ON DELETE CASCADE` here; perhaps just setting it null so we don't delete user data associated with the activity type being deleted.

Reply:
Changed to `on_delete=None`